Src/KBCsv.PerformanceTests/KBCsv.PerformanceTests.csproj

```diff
@@ -14,11 +14,16 @@
   <ItemGroup>
     <PackageReference Include="Moq" Version="4.7.8" />
     <PackageReference Include="xunit" Version="2.3.0-beta1-build3642" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\KBCsv\KBCsv.csproj" />
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0-beta1-build3642" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Any ideas what caused this `ItemGroup` to be added? Seems like it's related to VSTE? Are you able to run tests without it? Bit strange that it was added only to this project...